### PR TITLE
Fixed Travis-CI errors

### PR DESCRIPTION
**Pull request by matthewrsj**
Travis-CI now passes after implementing the following:

* Created a mock SMTP server and smtp-sink to accept email
* Mocked external method validate_email to return True when necessary

### CODE REVIEW

**Comment by matthewrsj** [comment]
@Kennric Can you review?

File: werkzeug_test.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import smtplib
 import unittest
 from request_handler import (Forms, create_msg, validate_name, is_valid_email,
-                             is_hidden_field_empty, is_valid_token)
+                     is_hidden_field_empty, is_valid_token)
```
Comment (Kennric) [inline werkzeug_test.py]: pep8 didn't warn on this indentation level? I think the previous indentation was more correct.

Comment (matthewrsj) [inline werkzeug_test.py]: @Kennric I don't know why that was changed. I will fix it.

File: werkzeug_test.py [hunk 2]
```diff
@@ -94,14 +96,17 @@ def test_validations_valid_data(self):
         """
         builder = EnvironBuilder(method='POST',
                                  data={'name': 'Valid Guy',
-                                       'email': 'example@osuosl.org',
+                                       'email': 'mrsj@osuosl.org',
```
Comment (Kennric) [inline werkzeug_test.py]: I thought we decided to go with all example emails and no real ones?

File: werkzeug_test.py [hunk 3]
```diff
@@ -188,10 +194,13 @@ def test_is_valid_email_with_valid(self):
         valid and exists. This function call should return true.
         """
         builder = EnvironBuilder(method='POST',
-                                 data={'email': 'example@osuosl.org'})
+                                 data={'email': 'mrsj@osuosl.org'})
```
Comment (Kennric) [inline werkzeug_test.py]: One more real email here

**Pull request merged**